cmake/sample_defs/targets.cmake

```diff
@@ -90,11 +90,14 @@ SET(TGT1_FILELIST cfe_es_startup.scr)
 
 # CPU2/3 are duplicates of CPU1.  These are not built by default anymore but are
 # commented out to serve as an example of how one would configure multiple cpus.
-#SET(TGT2_NAME cpu2)
-#SET(TGT2_APPLIST sample_app ci_lab to_lab sch_lab)
-#SET(TGT2_FILELIST cfe_es_startup.scr)
+
+# CPU2 is uncommented to test Travis CI's ability to build multiple targets, ensuring
+# the build system is functioning properly.
+SET(TGT2_NAME cpu2)
+SET(TGT2_APPLIST sample_app sample_lib ci_lab to_lab sch_lab)
+SET(TGT2_FILELIST cfe_es_startup.scr)
```

Comment:
Don't change the default behavior of cFE out of the box.  This should be part of CI if you want to check 2 targets with CI.